tests/test_load.py

```diff
@@ -260,7 +265,7 @@ def test_load(format, version):
     )
     db_root = db.meta['audb']['root']
 
-    assert audb.exists(DB_NAME, version=version)
+    assert audb.exists(DB_NAME, version=version, format=format)
```

Comment:
This was a bug in the tests before, we only checked if the version loaded with `format=None` was in the cache, but it should check for the actual requested format.

Reply:
I created https://github.com/audeering/audb/pull/251 to fix this independently from this pull request.

Reply:
#251 was merged.